### PR TITLE
Add Box Definitions for Uncompressed Codec

### PR DESCRIPTION
**Pull request by dukesook**
The _uncC_ & _cmpd_ boxes are defined in ISO/IEC 23001-17 and add support for uncompressed mp4 files.

The VisualSampleEntry codingname is "uncv".

### CODE REVIEW

File: src/parsing/uncC.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    this.component_count = stream.readUint16();
+    this.component_index = [];
+    this.component_bit_depth_minus_one = [];
+    this.component_format = [];
+    this.component_alight_size = [];
+    for (i = 0; i < this.component_count; i++) {
+        this.component_index.push(stream.readUint16());
+        this.component_bit_depth_minus_one.push(stream.readUint8());
+        this.component_format.push(stream.readUint8());
+        this.component_alight_size.push(stream.readUint8());
```
Comment (bradh) [inline src/parsing/uncC.js:13]:
```suggestion
        this.component_align_size.push(stream.readUint8());
```

File: src/parsing/uncC.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+BoxParser.createBoxCtor("uncC", function(stream) {
+    var i;
+    this.profile = stream.readUint32();
+    this.component_count = stream.readUint16();
+    this.component_index = [];
+    this.component_bit_depth_minus_one = [];
+    this.component_format = [];
+    this.component_alight_size = [];
```
Comment (bradh) [inline src/parsing/uncC.js:8]:
```suggestion
    this.component_align_size = [];
```

**Review by bradh — CHANGES_REQUESTED** [review]
Looks good, just a couple of typos.

**Comment by dukesook** [comment]
Thanks for catching those.

**Pull request merged**